data/pack/install.yml

```diff
@@ -0,0 +1,79 @@
+- name: Linux
+  methods:
+    - name: Arch Linux
+      content: |
+        On Arch Linux, there are two packages available:
+        
+        - [pack-cli](https://aur.archlinux.org/packages/pack-cli/)
+        - [pack-cli-bin](https://aur.archlinux.org/packages/pack-cli-bin/)
```

Comment:
How do you download from Arch Linux? Any CLI commands?

Reply:
For AURs, it's dependant on the package manager they use so we opted to be non-prescriptive.